ScanCommand/src/main/resources/cli-help/ballerina-scan.help

```diff
@@ -0,0 +1,69 @@
+NAME
+        ballerina-scan - Static code analyzer
+
+SYNOPSIS
+        bal scan [OPTIONS] [<package>|<source-file>]
+
+DESCRIPTION
+        Compiles and performs static code analysis, prints results to the console,  and reports results.
```

Comment:
```suggestion
        Compiles and performs static code analysis, prints results to the console, and reports results.
```

Reply:
Fixed in [da46b97](https://github.com/ballerina-platform/static-code-analysis-tool/pull/3/commits/da46b977ecd4bc8ac563fb490e97eed889b71f54)